applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -478,10 +478,105 @@ static int cmd_input(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
-SHELL_STATIC_SUBCMD_SET_CREATE(hw_codec_cmd,
-			       SHELL_COND_CMD(CONFIG_SHELL, input, NULL,
-					      " Select input\n\t0: LINE_IN\n\t\t1: PDM_MIC",
-					      cmd_input),
-			       SHELL_SUBCMD_SET_END);
+static int out1l_clear(void)
```

Comment:
Nice helper function. Suggest
```suggestion
static int analog_out1l_disconnect(void)
```